eng/pipelines/templates/steps/update_snippet.yml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  ServiceDirectory: ''
+  ValidateFormatting: false
+  EnvVars: {}
+
+steps:
+  - task: UsePythonVersion@0
```

Comment:
Do you think this is going to conflict with or override any other python version usage in the job that uses this?

Reply:
This script should work with any py3.

Can we specify something like use any python that is available?

Reply:
I don't think so with this task. This step mutates the global path so I just wanted to make sure it doesn't conflict with other steps in this job.

Reply:
We have same settings for tasks

run_pylint & run_black.

I guess this should be fine. :)

Reply:
As Scott suggested, changed it to use py 3.9